check_modulemd.py

```diff
@@ -220,6 +221,69 @@ def test_description_spelling(self):
             self.log.warn(
                 "Potential spelling problem in description: %s" % err.word)
 
+
+    def _read_license_file(self, filename):
+        """
+        Read all licenses from a file and return them as a list
+        """
+        try:
+            with open(filename) as f:
+                data = f.readlines()
+        except Exception as ex:
+            self.error("Could not open file %s (%s)" % (filename, ex))
+        licenses = []
+        for line in data:
+            #remove all comments from the file
+            license = re.sub("#.*","", line).strip()
+            if not license:
+                continue
+            licenses.append(license)
+        return licenses
+
+    def _split_license(self, license):
+        license_regex = re.compile(r'\(([^)]+)\)|\s(?:and|or|AND|OR)\s')
+        return (x.strip() for x in
+                (l for l in license_regex.split(license) if l))
+
+    def test_license(self):
+        """
+        Does the module provide license?
+        Based on code from: https://github.com/rpm-software-management/rpmlint/blob/master/TagsCheck.py
+        """
+        self.log.info(
+            "Checking for presence of license")
+        if not self.mmd.module_licenses:
+            self.error("No module license")
+
+        self.assertIsInstance(self.mmd.module_licenses, set)
+        self.assertGreater(len(self.mmd.module_licenses), 0)
+
+        valid_licenses = []
+        license_files = [
```

Comment:
Hard to tell whether modulemd and other files in the module dist-git repo are software, documentation or content. I think it could be all of that.